src/translations/en.json

```diff
@@ -2478,7 +2478,8 @@
               "setup_retry": "Retrying setup",
               "not_loaded": "Not loaded",
               "failed_unload": "Failed to unload"
-            }
+            },
+            "open_configuration_url": "Visit external configuration page"
```

Comment:
Maybe just visit configuration page? the icon shows its external

Reply:
Yea I agree. Visit device is short and sweet and covers most cases

Reply:
```suggestion
            "open_configuration_url": "Visit device"
```